core/xml.ts

```diff
@@ -545,6 +547,27 @@ export function appendDomToWorkspace(
  * @returns The root block created.
  */
 export function domToBlock(xmlBlock: Element, workspace: Workspace): Block {
+  const block = domToBlockInternal(xmlBlock, workspace);
+  if (workspace.rendered) renderManagement.triggerQueuedRenders();
+  return block;
+}
+
+/**
+ * Decode an XML block tag and create a block (and possibly sub blocks) on the
+ * workspace.
+ *
+ * This is defined internally so that it doesn't trigger an immediate render,
+ * which we do want to happen for external calls.
+ *
+ * @param xmlBlock XML block element.
+ * @param workspace The workspace.
+ * @returns The root block created.
+ * @internal
+ */
+export function domToBlockInternal(
```

Comment:
I get why you made `domToBlock` render immediately, but I don't love this name. It's not very descriptive, and it implies external developers would never want to call it. But in reality they might be perfectly fine not triggering an immediate render. What about `domToBlockAsync`? or something better than that in the same vein? Then we don't have to mark it as internal.

Or is your plan to eventually make the breaking change to `domToBlock`? in that case i don't care quite as much if this method is intended to be temporary

Reply:
My plan is indeed to eventually make the breaking change to `domToBlock` (unless there's an important reason we don't want to do that)! I chose the `domToBlockInternal` name to match the naming of the JSON serialization systsem.